test/functional/cursor.test.js

```diff
@@ -1550,6 +1550,50 @@ describe('Cursor', function () {
     }
   });
 
+  it('does not auto destroy streams', {
+    metadata: {
+      requires: { topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'] }
```

Comment:
I know this `topology` setting is what the other tests have, but for some time the valid topologies have been paired down to the first three in this list: https://github.com/mongodb/node-mongodb-native/blob/a516903bf91e34bb83a3bf565c2a39cdbfddd072/test/tools/runner/filters/mongodb_topology_filter.js#L47-L55

So you can omit this filter all together

Reply:
Oh thanks for that - I've removed now.